plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -309,14 +270,39 @@ library PaymentStartInFlightExit {
 
     function setInFlightExitInputs(
         PaymentExitDataModel.InFlightExit storage ife,
-        PaymentTransactionModel.Transaction[] memory inputTxs,
-        UtxoPosLib.UtxoPos[] memory inputUtxosPos
+        StartExitData memory exitData
     )
         private
     {
-        for (uint i = 0; i < inputTxs.length; i++) {
-            uint16 outputIndex = inputUtxosPos[i].outputIndex();
-            ife.inputs[i] = inputTxs[i].outputs[outputIndex];
+        for (uint i = 0; i < exitData.inputTxs.length; i++) {
+            uint16 outputIndex = exitData.inputUtxosPos[i].outputIndex();
+            WireTransaction.Output memory output = WireTransaction.getOutput(exitData.inputTxs[i], outputIndex);
+
+            ife.inputs[i] = PaymentExitDataModel.WithdrawData(
+                output.outputGuard,
+                exitData.outputIds[i],
+                output.token,
+                output.amount
+            );
+        }
+    }
+
+    function setInFlightExitOutputs(
+        PaymentExitDataModel.InFlightExit storage ife,
+        StartExitData memory exitData
+    )
+        private
+    {
+        for (uint i = 0; i < exitData.inFlightTx.outputs.length; i++) {
+            bytes32 outputId = OutputId.computeNormalOutputId(exitData.inFlightTxRaw, i);
```

Comment:
I would add a comment that deposit tx cannot in-flight exit.

Reply:
done